support/openharmony/build-profile.json5

```diff
@@ -0,0 +1,44 @@
+{
+  "app": {
+    "products": [
+      {
+        "name": "default",
+        "signingConfig": "default",
+        "compileSdkVersion": 11,
+        "compatibleSdkVersion": 11,
+        "targetSdkVersion": 11,
+        "runtimeOS": "OpenHarmony"
+      },
+      {
+        "name": "harmonyos",
+        "signingConfig": "default",
+        "compatibleSdkVersion": "4.0.0(10)",
```

Comment:
let's change this to "4.1.0(11)" to match the API version for OpenHarmony.

Reply:
Good point. I've changed it to "4.1.0(11)".